src/Component/FieldContainer/FieldContainer.tsx

```diff
@@ -27,64 +27,32 @@
  */
 
 import * as React from 'react';
-import { Input, Form } from 'antd';
 
-import './TitleField.less';
+import './FieldContainer.less';
+
+import { localize } from '../LocaleWrapper/LocaleWrapper';
```

Comment:
Do we actually need `localize` on this component? There is nothing to translate, right?